client/settings/general-settings-section/payment-method.js

```diff
@@ -138,7 +137,7 @@ const PaymentMethod = ( { method, data } ) => {
 			// eslint-disable-next-line camelcase
 			wc_stripe_settings_params.has_klarna_gateway_plugin );
 
-	const isDisabledButChecked = PAYMENT_METHOD_CARD === method && isOCEnabled;
+	const isDisabledButChecked = false;
```

Comment:
Should we just remove this variable, and the `disabledButChecked` prop below it? It looks like it no longer has any use case for it.